test-cypress/integration/setup/connectors-lucene.spec.js

```diff
@@ -0,0 +1,153 @@
+    const connectorDeleteToastMessage = 'Deleted connector ' + luceneConnectorName;
+
+    before(() => {
+        repositoryId = 'repo' + Date.now();
+        cy.createRepository({id: repositoryId});
+        cy.presetRepositoryCookie(repositoryId);
+        cy.visit('/connectors');
+    });
+
+    after(() => {
+        cy.deleteRepository(repositoryId);
+    });
+
+    it('Test that no indexes are available', () => {
+        getConnectorsPage()
+            .should('be.visible')
+            .and('contain', 'No connector instances.');
+    });
+
+    it('Create a lucene connector', () => {
+        getNewLuceneConnectorButton()
+            .click();
+        getCreateLuceneConnectorPage()
+            .should('be.visible')
+            .and('contain', 'Create new Lucene Connector');
+        getConnectorNameField()
+            .find('input')
+            .type(luceneConnectorName);
+        getFieldNameField()
+            .find('input')
+            .type(fieldName, {force:true});
+        getPropertyChainField()
+            .find('input')
+            .type(connectorPropertyChain, {force:true});
+        getUriTypes()
+            .find('input')
+            .type(uriType);
+        getOkButton()
+            .click();
+        getConnectorCreationDialog()
+            .should('be.visible')
+            .and('contain', luceneConnectorName);
+        getConnectorStatusToastMessage()
+            .should('be.visible')
+            .and('contain', connectorCreateToastMessage);
+    });
+
+    it('Test connector copy functionality', () => {
+        getConnectorsPage()
+            .should('be.visible');
+        getConnectorInstance(0)
+            .find('.icon-copy')
+            .should('be.visible')
+            .click();
+        getOkButton()
+            .click({force:true});
+        getConnectorCreationDialog()
+            .should('be.visible')
+            .and('contain', luceneConnectorName + '-copy');
+        getConnectorStatusToastMessage()
+            .should('be.visible')
+            .and('contain', connectorCreateToastMessage + '-copy');
+    });
+
+    it('Test connector delete functionality', () => {
+        getConnectorInstance(1)
+            .find('.icon-trash')
+            .should('be.visible')
+            .click();
+        getConfirmConnectorDeletebutton()
+            .should('be.visible')
+            .click();
+        getConnectorStatusToastMessage()
+            .should('be.visible')
+            .and('contain', connectorDeleteToastMessage + '-copy');
+    });
+
+    function getConnectorsPage() {
+        return cy.get('.connectorsPage');
+    }
+
+    function getNewLuceneConnectorButton() {
+        return getConnectorsPage().find('.new-connector-btn');
+    }
+
+    function getCreateLuceneConnectorPage() {
+        return cy.get('.modal-content');
+    }
+
+    function getConnectorNameField() {
+        return getCreateLuceneConnectorPage().find('.connector-name-field');
```

Comment:
Better move the `input` selector in here to encapsulate it fully e.g.
```javascript
 function getConnectorNameField() {
        return getCreateLuceneConnectorPage().find('.connector-name-field input');
}
```
and use it like
```javascript
        getConnectorNameField()
            .type(luceneConnectorName);
```
Same applies to the other form fields: name and property chain

Reply:
Fixed with commit fe274051f6f1c19f89ac97e134e6086d2d98512c

Reply:
This is also applicable to the other fields.

Reply:
Fixed in commit 331d9c9122e4c702a054325ddc791ec429a005dd